src/main/java/Parser.java

```diff
@@ -0,0 +1,82 @@
+import exceptions.EmptyNameException;
+import exceptions.InvalidTaskIndexException;
+import exceptions.NoTasksException;
+import exceptions.UnknownCommandException;
+import objects.Task;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Parser {
+    public enum Command {
+        BYE, LIST, MARK, UNMARK, TODO, DEADLINE, EVENT, DELETE, FIND
+    }
+
+    private static Ui ui = new Ui();
+    private static TaskList taskList = new TaskList();
+    private static Scanner input = new Scanner(System.in);
+
+    public static Scanner getScanner() {
+        return input;
+    }
```

Comment:
I think this function is currently unused, perhaps remove it? Or if not, might want to add header comment to it too.